content/02.intro.md

```diff
@@ -5,23 +5,23 @@ A systematic review of ML applications in rare diseases (as defined in the Europ
+Supervised methods can learn correlations of features (e.g., expression measurements of a large number of genes) that may be associated with these labels to predict or infer these labels in unlabeled data, such as predicting which patients will or will not respond to treatment. 
+Therefore, if a study aims to classify patients with a rare disease into disease subtypes based on high-throughput molecular profiling, a supervised ML algorithm is appropriate to carry out this task. 
+Conversely, unsupervised learning algorithms learn patterns or features from unlabeled data. 
+In the absence of known disease subtypes, unsupervised ML approaches can be applied to gene expression data to identify groups of samples with similar patterns of molecular states or pathway activity.  [@doi:10.1158/0008-5472.CAN-08-2100].
 Unsupervised approaches can also extract combinations of features (e.g., genes) that may describe a certain cell type or pathway.
 See Box 1 for more examples of how ML can be used in rare disease research.
 
 While ML can be a useful tool, there are challenges in applying ML to rare disease datasets. 
-ML methods are generally most effective when using large datasets; thus analyzing high dimensional biomedical data (i.e. data with typically > 1000 features, e.g. 20,000 genes) from rare diseases datasets that typically contain 20 to 99 samples is challenging[@https://www.fda.gov/media/99546/download; @doi:10.1186/s13023-020-01424-6].
-Small datasets tend to lack statistical power and magnify the susceptibility of ML to misinterpretation and unstable performance. 
-With insufficient data, an unsupervised model will fail to identify patterns that are useful for biological discovery (i.e. “perform” poorly). 
-Similarly, supervised models require datasets where the phenotype labels have very little uncertainty (or “label-noise”) [@doi:10.1093/jamia/ocw028] – termed "gold standard" datasets. 
-Datasets with high label-noise decrease prediction accuracy and necessitate larger sample sizes during training [@doi:10.1109/tnnls.2013.2292894].
+ML methods are generally most effective when using large datasets; analyzing high dimensional biomedical data such as gene expression with many thousands of features from rare disease datasets that typically contain relatively few samples is challenging[@https://www.fda.gov/media/99546/download; @doi:10.1186/s13023-020-01424-6].
+Small sample datasets tend to lack statistical power and magnify the susceptibility of ML to misinterpretation and unstable performance. 
+With insufficient data, an unsupervised model will fail to identify patterns that are useful for biological discovery. 
+In the case of supervised models, modelsthey can be adversely impact if sample labels are uncertain or contain “label-noise”. [@doi:10.1093/jamia/ocw028] 
+Datasets with high label-noise decrease prediction accuracy and necessitate larger sample sizes during model training [@doi:10.1109/tnnls.2013.2292894].
```

Comment:
```suggestion
Datasets with high label-noise decrease prediction accuracy and necessitate larger sample sizes during model training [@doi:10.1109/tnnls.2013.2292894] (See Box 2).
```

Reply:
In response to Justin's comment: This is at the first time I’ve seen “training” mentioned. Should be defined